src/screens/CreatePost/elements/PollItem.js

```diff
@@ -63,7 +73,7 @@ const S = StyleSheet.create({
     borderRadius: 10,
     marginVertical: 4,
     paddingHorizontal: 8,
-    borderColor: COLORS.holyTosca
+    borderColor: colors.holytosca,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The borderColor property value has been changed. Make sure that `colors.holytosca` exists and is the intended color.
```diff
- borderColor: COLORS.holyTosca
+ borderColor: colors.holytosca,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->